src/main/java/com/woowacourse/caffeine/support/SearchParser.java

```diff
@@ -0,0 +1,24 @@
+package com.woowacourse.caffeine.support;
+
+import com.woowacourse.caffeine.application.dto.ShopSearchDto;
+import com.woowacourse.caffeine.application.exception.InvalidSearchRequestException;
+
+public class SearchParser {
+    private static final String SEARCH_DATA_SPLIT_DELIMITER = ",";
+    private static final String SEARCH_KEY_VALUE_SPLIT_DELIMITER = "%3D";
```

Comment:
아래와 같이 표현하면 어떤 값인지 한눈에 알 수 있지 않을까요?
```suggestion
    private static final String SEARCH_KEY_VALUE_SPLIT_DELIMITER = URLEncoder.encode("=", "utf-8");
```

Reply:
오 이런방법이있었군요~

Reply:
혹시 이 부분 실제로 돌려서 확인이 된걸까요??

실제로 돌려서 해보니..  스프링에서 알아서 '=' 로 바꿔서 
```java
public Page<ShopResponse> search(final String query, final Pageable pageable) 
``` 
여기에 query 에 채워주는 것 같아서요.

'query=keyWord%3Daddress,contents%3D오금로' 인 경우
query 에는 'keyWord=address,contents=오금로' 이렇게 들어오는 것 같아요

Reply:
![image](https://user-images.githubusercontent.com/40996389/71452441-7bab7c80-27c8-11ea-9d81-41e44d4b63a1.png)
일단 디버깅 할 때에는 이렇게 들어오긴 해요